Licensing-strategy.md

```diff
@@ -0,0 +1,25 @@
+# Licensing strategy
+
+* [Project License](#project-license)
+* [License file](#license-file)
+* [License for individual files](#license-for-individual-files)      
+
+## Project License
+
+The license for the [Kata Containers](https://github.com/kata-containers)
+project is [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0).
+
+## License file
+
+All repositories in the project have a top level file called `LICENSE`. This
+file lists full details of all licences used by the repository.
+
+## License for individual files
+
+Where possible all files in all repositories also contain an
```

Comment:
Changing "and" to "a": Where possible all files in all repositories also contain a

Reply:
Ah, you mean "an" -> "a" ;) Done.